### PR TITLE
Add GPS filtering to task creation flow

### LINKED ISSUE #82: Image EXIF GPS task filter
## Feature description and context
The two primary ways that lab leads segment their data is through date and location. We already provide date, so now we should provide location.

## Feature sign-off requirements
- New task > Select Image modal has Latitude and Longitude fields, displaying in decimal degrees
- Latitude
  - range is -90 to 90
  - placeholder text is from -90 to 90
- Longitude
  - range is -180 to 180
  - placeholder text is from -180 to 180
- images that are missing latitude or longitude are filtered from any range set in the respective field
- range works with a single input. if user inputs Latitude From 30 To (blank), then range is treated as 30-90

## Design
_Design is general ux; do not implement pixel perfect recreation. use patterns/styles as currently implemented on page_

image

### PR DESCRIPTION
**Pull request by ejcunningham**
Add support to filter images by GPS location during task creation flow

PR fixes #82

**Changes**
- Add latitude+longitude filters to the task creation form
- Change Image model to default location attributes to invalid values rather than using `required=false`, which seems to be the same as defaulting to 0 (a valid value) rather than `undefined`
- Plumb filters through the task POST endpoint and taskQueuer

Note the ticket specifies setting a placeholder text on the form fields, but placeholders appear to be broken on this whole form. Suggest we fix that bug as a separate task.

Test Plan:
- Tested that the UI can correctly reflect filters in image-count
- Verified taskQueuer finds the right files
- Verified image preview shows the right files
- Verified ground truth page and CSV exports still work properly with model changes

Screenshot 2024-06-13 at 7 41 25 AM

### CODE REVIEW

File: api/models/Images.js [hunk 1]
```diff
@@ -55,55 +57,102 @@ module.exports = {
...
+  },
 
-        // all these things reference Images, so must be also deleted where appropriate
-        console.info('deleting associated objects for imageId=%s', id);
-        await QueuedImages.deleteForImage(id);
-        await Annotations.deleteForImage(id);
-        await GroundTruths.deleteForImage(id);
-        await LineDivisions.deleteForImage(id);
-        await SequencedPairs.deleteForImage(id);
+  deleteCascade: async function(id) {
```
Comment (ejcunningham) [inline api/models/Images.js:107]: Note there are no changes below this point, I just fixed the inconsistent indentation

**Review by naknomum — COMMENTED** [review]
generally looks correct to me, except i am not sure if the original issue means the user should be prevented (or warned?) if they enter values outside of the valid ranges. might be a question for @TanyaStere42 to confirm the answer on.

for example, if a user enters a **latitude** value of `100`, should they get a warning/error that they are out of range?

**Comment by TanyaStere42** [comment]
hiya @naknomum ! Yes, the intention of the ticket is that values outside of the lat/long ranges listed should be disallowed from being entered.

**Review by naknomum — APPROVED** [review]
per updated info (below), i am approving this now.

> I talked to the customer and it seems like that validation is important but not blocking. We can go ahead and merge assuming there was nothing else in the code review, and I'll make a new ticket for the form validation (@TanyaStere42)

**Pull request merged**